src/program.ts

```diff
@@ -54,9 +54,9 @@ program
       if (options.port) {
         const sessions = new Map<string, SSEServerTransport>();
         const httpServer = http.createServer(async (req, res) => {
+          const url = new URL(req.url ?? '', `http://${req.headers.host}`);
```

Comment:
```suggestion
          const url = new URL(`http://${req.headers.host ?? 'localhost'}${req.url || ''}`;);
```
what about this?

Reply:
this should be exactly the same, because `req.url` is always absolute.